winmdgenerator: 0.2.185-preview nuget is broken

MetadataTasks.dll is indeed missing https://nuget.info/packages/Microsoft.Windows.WinmdGenerator/0.2.185-preview

```
Sdk.targets(19,5): error MSB4062: The "MetadataTasks.CompileIdls" task could not be loaded from the assembly
C:\Users\Rafael\.nuget\packages\microsoft.windows.winmdgenerator\0.2.185-preview\tools\net5.0\MetadataTasks.dll.

Could not load file or assembly
'C:\Users\Rafael\.nuget\packages\microsoft.windows.winmdgenerator\0.2.185-preview\tools\net5.0\MetadataTasks.dll'.

The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly
and all its dependencies are available, and that the task contains a public class that implements
Microsoft.Build.Framework.ITask.
```

Please re-open, this is still broken. @AArnott @sotteson1 

```
error MSB4236: The SDK 'Microsoft.Build.NoTargets' specified could not be found.
```

--- a/sources/GeneratorSdk/sdk/sdk.props
+++ b/sources/GeneratorSdk/sdk/sdk.props
@@ -4,7 +4,7 @@
     <MicrosoftNoTargetsSdkImported Condition="'$(UsingMicrosoftNoTargetsSdk)' != 'true'">true</MicrosoftNoTargetsSdkImported>
   </PropertyGroup>
 
-  <Import Condition="'$(UsingMicrosoftNoTargetsSdk)' != 'true'" Sdk="Microsoft.Build.NoTargets" Project="sdk.props"/>
+  <Import Condition="'$(UsingMicrosoftNoTargetsSdk)' != 'true'" Sdk="Microsoft.Build.NoTargets" Version="3.0.4" Project="sdk.props"/>
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>

--- a/sources/GeneratorSdk/sdk/sdk.targets
+++ b/sources/GeneratorSdk/sdk/sdk.targets
@@ -63,5 +63,5 @@
     <RemoveDir Directories="obj;bin" />
   </Target>
 
-  <Import Condition="'$(MicrosoftNoTargetsSdkImported)' == 'true'" Sdk="Microsoft.Build.NoTargets" Project="sdk.targets"/>
+  <Import Condition="'$(MicrosoftNoTargetsSdkImported)' == 'true'" Sdk="Microsoft.Build.NoTargets" Version="3.0.4" Project="sdk.targets"/>
 </Project>

